db/migrate/20211028083305_change_column_type_of_reviews.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeColumnTypeOfReviews < ActiveRecord::Migration[6.1]
+  def change
+    change_column :reviews, :review_date, :string
```

Comment:
Why is this a string instead of datetime?

Reply:
-- comma separated list of dates is added. A hash would have been preferable here...